kubernetes/bosh-cloud-config.yml

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
These ports need to be unique for the whole cluster. And the range is even configurable. We might force people to edit bosh-deployment by hard coding them. I was thinking about adding a minikube ops file. That could actually hard-code all the ports for ease of first usage. What do you think?